src/samtools/idxstats/config.vsh.yaml

```diff
@@ -0,0 +1,52 @@
+name: samtools_idxstats
+namespace: samtools
```

Comment:
Move this component to `src/<namespace>/<name>`; that is, `src/samtools/samtools_idxstats`.